docs/hosting/configuration/configuration-examples/prometheus.md

```diff
@@ -23,7 +23,7 @@ Both `main` and `worker` instances are able to expose metrics.
 
 To enable queue metrics, set the `N8N_METRICS_INCLUDE_QUEUE_METRICS` env var to `true`. You can adjust the refresh rate with `N8N_METRICS_QUEUE_METRICS_INTERVAL`.
 
-Queue metrics are only available for the `main` instance in single-main mode.
+These metrics are gathered from Bull and exposed by main instances. On a multi-main setup, when aggregating queries you can identify the leader using the gauge `instance_role_leader`, set to `1` for the leader main and `0` otherwise.
```

Comment:
```suggestion
n8n gathers these metrics from Bull and exposes them on the main instances. On multi-main setups, when aggregating queries, you can identify the leader using the `instance_role_leader` gauge, set to `1` for the leader main and `0` otherwise.
```